src/Admin/Extension/LockExtension.php

```diff
@@ -59,10 +59,7 @@ public function configureFormFields(FormMapper $form)
 
             $form->add(
                 $fieldName,
-                // NEXT_MAJOR: remove the check and add the FQCN
-                method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')
-                    ? 'Symfony\Component\Form\Extension\Core\Type\HiddenType'
-                    : 'hidden',
+                'Symfony\Component\Form\Extension\Core\Type\HiddenType',
```

Comment:
use HiddenType::class for every FQCN

Reply:
I assumed we would replace it for the whole project in separate PR

Reply:
There are no cs-fixer or similar to automate it, as far as we know. So we are changing it on the lines that the people modify on their PR

Reply:
Will be enough make global replace by regular expression (f.e. `'\w+\\[\w\\]+'`).
I can make it when this PR will be done.

Reply:
please do